src/_locales/en_US/messages.json

```diff
@@ -508,5 +508,41 @@
                 "example": "Facebook Like"
             }
         }
+    },
+    "sync_intro": {
+        "message": "Cloud sync:<ul><li>Requires Firefox/Chrome Sync</li><li>Upload overwrites any existing Privacy Badger data in the cloud</li><li>Download combines the lists of sites where your Badger is disabled</li></ul>",
+        "description": "A brief explanation of how syncing works. Shown above the upload/download cloud data buttons under the Manage Data options page tab."
+    },
+    "upload": {
+        "message": "Upload",
+        "description": "Button label for uploading data to the cloud on Manage Data options page tab."
+    },
+    "upload_cloud": {
+        "message": "Export disabled sites to cloud",
+        "description": "Button title for uploading data to the cloud on Manage Data options page tab."
+    },
+    "download_cloud": {
+        "message": "Import disabled sites from cloud",
+        "description": "Button title for downloading data from the cloud on Manage Data options page tab."
+    },
+    "download_cloud_success": {
+        "message": "Cloud data imported successfully.",
+        "description": "Message displayed to the user after successfully merging cloud data."
+    },
+    "download_cloud_failure": {
+        "message": "Cloud data could not be downloaded.",
+        "description": "Message displayed to the user after failing to download cloud data."
+    },
+    "upload_cloud_success": {
+        "message": "Cloud data uploaded successfully.",
+        "description": "Message displayed to the user after successfully uploading local data."
+    },
+    "upload_cloud_failure": {
+        "message": "Cloud data could not be uploaded.",
+        "description": "Message displayed to the user after failing to upload local data."
+    },
+    "download_cloud_no_data": {
```

Comment:
I never got this message to trigger -- before I uploaded anything, the download button gave the "download_cloud_failure" message, and after uploading empty data, it gave the "success" message. See comment in `options.js`.

Reply:
To get this message, something has to go wrong writing to the synced storage (network connectivity loss during write might do it, as would going over quota which is 100kB total, 8kB per item [with the list of disabled sites being 1 item] see https://developer.chrome.com/apps/storage#properties)

Reply:
This user data file I manipulated to have too long of a disabled sites list, and produces the error condition `QUOTA_BYTES_PER_ITEM quota exceeded`.
[PrivacyBadger_user_data-long-DL.zip](https://github.com/EFForg/privacybadger/files/2350817/PrivacyBadger_user_data-long-DL.zip)